### ISSUE TITLE
Emscripten troubles "GET /installHook.js.map HTTP/1.1" 404 -"

### ISSUE THREAD

**Issue by stanybebe** [openframeworks/openFrameworks#8219]
Hey there sweet people! I am having trouble with this tutorial here :
https://openframeworks.cc/setup/emscripten/

i have downloaded the nightly builds v0.12.0 and also I am using the newest build of emscripten SDK version '3.1.72' 

also
   node-20.18.0-64bit
   python-3.9.2-64bit

and I can build my project and it creates the em folder  but on load i keep getting this error i cannot find anything about online which i guess was supposed to have been generated:
"GET /installHook.js.map HTTP/1.1" 404 -
"GET /index.wasm.map HTTP/1.1" 404 -

the headers in emscripten make file "CXX = -gsource-map" only create the
index.wasm.map. Any help would be greatly appreciated!

**Comment by ofTheo** [comment]
@Jonathhhan - could you take a look at this? 
Both 0.12.0 and nightly are failing for me with the latest Emscripten version.

**Comment by Jonathhhan** [comment]
@ofTheo I will take a look and give it a try (but cannot promise to solve it). Some months ago I had a similar issue and I could not solve it, too. It was also compiling, but not running (not sure about the exact error anymore). Thats because of my lack in knowledge about web technologies...

**Comment by stanybebe** [comment]
Is there a build and emscripten version we know works?

**Comment by Jonathhhan** [comment]
@stanybebe yes. 3.1.50 and OF Nightly should work: https://forum.openframeworks.cc/t/emscripten-on-osx-not-working-for-me/28911/13

**Comment by stanybebe** [comment]
hm now getting a new error " GLctx is not defined" on build


Uncaught TypeError: GLctx is undefined
    _glGetString http://localhost:6931/index.js:1
    callRuntimeCallbacks http://localhost:6931/index.js:1
    initRuntime http://localhost:6931/index.js:1
    doRun http://localhost:6931/index.js:1
    run http://localhost:6931/index.js:1

**Comment by stanybebe** [comment]
really struggling with this one, no matter what emscripten flags or version and making sure i remove all previous objs, i cannot get past `Uncaught TypeError: GLctx is undefined`, seems like in the wasm its pointing to 

```
ofGLESVersionFromGL() http://localhost:6931/index.wasm:365332
    ofTexture::ofTexture() http://localhost:6931/index.wasm:301849
```
    how should my main.h look for emscripten?

**Comment by ofTheo** [comment]
Just to cross post this. 
These are all the issues in getting the nightly to run: 

for the libfmt issue - remove: 
`of_v20241205_osx_release/libs/fmt/lib/emscripten` fixes this 

But I get errors relating to wasm after that so need to install 3.1.73 ( to match apothecary ): 
```
./emsdk install 3.1.73
./emsdk activate 3.1.73
```

Then stuff gets built now in a bin/em/ folder so that breaks running it following the normal methods. 

So you have to run like:
` emrun bin/em/3DPrimitivesExample/index.html `

Then we get the gl context runtime error. 

<img width="522" alt="image" src="https://github.com/user-attachments/assets/cf54a1dd-8103-49f8-966b-60beaa6d3d66">

**Comment by dimitre** [comment]
it seems the same function addressed in #8212

**Comment by Jonathhhan** [comment]
@ofTheo I tried the latest OF / Emscripten and it is failing for me, too (same error as yours).
